packages/authhero/test/routes/universal-login/code.spec.ts

```
@@ -117,7 +117,7 @@ describe("code", () => {
     expect(reUsedCode).toContain("Login session closed");
   });
 
-  it("should create a new account", async () => {
+  it.only("should create a new account", async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)